frontend/00-browsertrix-nginx-init.sh

```diff
@@ -13,6 +13,6 @@ else
 fi
 
 mkdir -p /etc/nginx/resolvers/
-echo resolver $(awk 'BEGIN{ORS=" "} $1=="nameserver" {print $2}' /etc/resolv.conf) valid=10s ipv6=off";" > /etc/nginx/resolvers/resolvers.conf
+echo resolver $(grep -Eo '([0-9]{1,3}\.){3}[0-9]{1,3}' /etc/resolv.conf) valid=10s ipv6=off";" > /etc/nginx/resolvers/resolvers.conf
```

Comment:
```suggestion
echo resolver $(awk 'BEGIN{ORS=" "} $1=="nameserver" {print $2}' /etc/resolv.conf) valid=10s";" > /etc/nginx/resolvers/resolvers.conf
```

Per comments, maybe should remove ipv6=off instead.

Reply:
@Schmoaaaaah does this work as a better option? If so, lets go with this.

Reply:
that one wouldnt work as we need [] around ipv6 addresses.
```bash
grep -oP '(?<=nameserver\s)[^\s]+' /etc/resolv.conf | awk '{ if ($1 ~ /:/) { printf "[" $1 "] "; } else { printf $1 " "; } }'
```
this would work.

Reply:
Do you mind update the PR to that, and we can add it.